io.vikunja.Vikunja.yml

```diff
@@ -0,0 +1,38 @@
+app-id: io.vikunja.Vikunja
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+command: vikunja-desktop
+
+finish-args:
+  - --socket=x11
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - --env=ZYPAK_SANDBOX_FILENAME=chrome-sandbox
+
+modules:
+  - name: vikunja
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/bin/extra /app/lib/extra
+      - mv *.so* /app/lib/
+      - install -D vikunja.sh "/${FLATPAK_DEST}/bin/vikunja"
+      - mv * /app/bin/extra/
```

Comment:
I still so no .desktop. appstream or icon installed in the right locations.

Reply:
This is the reason the build fails right now. I have lost count of how many times I said this is not done.  But there is something but trying the same thing over and over and expecting a different result. It won't work.

Reply:
I will fix it now